### ISSUE TITLE
Support for Homey Pro (Early 2023) 

### ISSUE THREAD

**Issue by leonoosterhuis** [Joolee/nl.joolee.homey.espeasy#38]
Hi there, is it possible to provide support for the new homey coming out in february?

Or is this a lot of update work required? I am always available to support you in this.

I also speak Dutch if that is easier

**Comment by tjarb** [comment]
Hi There, I'm also interested in support for Homey pro (2023). 
Also Dutch :-)

**Comment by FreeNorway** [comment]
Espeasy is great, i hope for an update for homey2023. I can contribute with translation to Norweigan (from English) If wanted, but i cannot partisipate in updating as i have no clue about programming. What amazed me about the app is how Easy you made it for me to enter the world of esp so i thank you for that. People like me with almost no knowledge and not so much patiens can actually make affordable working solutions to automate stuff and save electricity. I have some fun ideas i want to try out with esp but i dont have the comfidense yet to go ahed whitout an app that compensates for stupid so i really hope you find the time to develop your app further.  Greetings from Norway

**Comment by Joolee** [comment]
Hi all! Sorry for not providing any updates lately. I’ve been working the the last few days on updating the app to SDK-V3 (which will make it work on the new 2023 Homey’s) but I can’t make any promises for the timeline.
So far it has not been too complicated but I have no idea what horrors might still be waiting for me.

**Comment by Joolee** [comment]
> Espeasy is great, i hope for an update for homey2023. I can contribute with translation to Norweigan (from English) If wanted, but i cannot partisipate in updating as i have no clue about programming. What amazed me about the app is how Easy you made it for me to enter the world of esp so i thank you for that. People like me with almost no knowledge and not so much patiens can actually make affordable working solutions to automate stuff and save electricity. I have some fun ideas i want to try out with esp but i dont have the comfidense yet to go ahed whitout an app that compensates for stupid so i really hope you find the time to develop your app further. Greetings from Norway

You can download the language file for English, linked below. The file format is json which isn't too hard to work with. I'd prefer if you keep the format in valid json, which you can check in a number of editors (just Google for 'json formatter' or something)
Note: Variables are the words with __ before and after. \n makes a new-line and I've used some simple HTML in a few strings. If you want to use a double quote like ", you'll need to escape it with a backslash: \\"

Just make a new issue with the json in the post (preferably in a code-block: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) and I'll include it in the next update.

https://github.com/Joolee/nl.joolee.homey.espeasy/blob/main/locales/en.json

**Comment by tjarb** [comment]
Awesome! Yes well, life of a programmer is rarely sunshine and rainbows. Thanks for spending your valuable time.
I see you already have the Dutch language, otherwise I would have translated.

**Comment by Tequila329** [comment]
Hi, 

is there any update regarding the Homey 2023 implementation? 

Thanks and best regards

**Comment by tjarb** [comment]
@Tequila329 
Last change / commit was 9th of June, and that was a small fix. So no, not really.

**Comment by astrapowerrr** [comment]
Can’t wait to see this app live in the homey AppStore.

**Comment by lenstar** [comment]
Any update for the homey pro 2023?

**Comment by luccie007** [comment]
Yeah, any update? I use it for my pool controller, and summer is coming :P  need any help?

**Comment by basnahuis** [comment]
Please update the app for homey 2023

**Comment by rkorpelainen** [comment]
Hey,
Any plan for update to support Homey 2026 Pro? 
Would be much appreciated.
